app/src/main/java/com/portfolio/tasky/entry/models/AuthenticationModel.kt

```diff
@@ -0,0 +1,11 @@
+package com.portfolio.tasky.entry.models
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+data class AuthenticationRequest(@Expose @SerializedName("email")var email : String?,
+                                 @Expose @SerializedName("password")var password : String?)
+
+data class AuthenticationResponse(@Expose @SerializedName("token")var token : String?,
+                                  @Expose @SerializedName("userId")var userId : String?,
+                                  @Expose @SerializedName("fullName")var fullName : String?)
```

Comment:
Why are these nullable?

Request will always be non null since you'll create it based on user input